docs/docs/run/6_registry.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Registry
+sidebar_label: Registry
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+## Creating a private repository for your image
+
+We provide a private image registry you can push images to with each service. To make use of it you can start by creating a service and configuring it:
+
+![configure service](/img/run/registry_1.png)
+
+Note that we are leaving the Image empty, this will autopopulate the service with the provisioned registry. We are also setting `Replicas` to 0 to avoid starting the service and incur costs before we have pushed the image.
```

Comment:
```suggestion
Note that we are leaving the Image empty, this will auto-populate the service with the provisioned registry. We are also setting `Replicas` to 0 to avoid starting the service and incurring costs before we have pushed the image.
```